community/io/src/main/java/org/neo4j/io/fs/DefaultFileSystemAbstraction.java

```diff
@@ -133,8 +135,13 @@ public void deleteRecursively( File directory ) throws IOException
     }
 
     @Override
-    public boolean renameFile( File from, File to ) throws IOException
+    public boolean move( File from, File to, CopyOption... copyOptions ) throws IOException
     {
+        if ( copyOptions.length > 0 )
+        {
+            Files.move( from.toPath(), to.toPath(), copyOptions );
+            return true; // will throw if failure
+        }
         return FileUtils.renameFile( from, to );
```

Comment:
It's ugly indeed, but I selected this implementation after looking at the even uglier windows specific bug-fix in `FileUtils.renameFile`. I think the full move over to `Files.move` should happen when more time is available to handle the potential bugs.